spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -268,17 +650,349 @@ public PublisherBuilder<IncomingMessage, OutgoingMessage> processMessages() {
 
 ==== Message acknowledgement
 
+Acknowledgement is an important part of message processing.
 Messages are either acknowledged explicitly, or implicitly by the implementation.
 
-===== Explicit acknowledgement
+Acknowledgement is controlled by the `org.eclipse.microprofile.reactive.messaging.Acknowledgment` annotation. 
+The annotation allows configuring the acknowledgement strategy among:
+
+* `NONE` - no acknowledgment is performed
+* `MANUAL` - the user is responsible for the acknowledgement, so the Reactive Messaging implementation does not apply implicit acknowledgement
```

Comment:
the user is responsible for the acknowledgement, by calling the `Message#ack()` method,  so the..